swagger/oauth.yaml

```diff
@@ -80,6 +85,8 @@ definitions:
     properties:
       access_token:
         type: "string"
+      refresh_token:
+        type: "string"
```

Comment:
`refresh_token` is only sometimes provided in response values.  Is there a way to indicate optional values here?